CHANGELOG.md

```diff
@@ -2,6 +2,69 @@
 
 Certbot adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.25.0 - 2018-06-06
+
+### Added
+
+* Support for the ready status type was added to acme. Without this change,
+  Certbot and acme users will begin encountering errors when using Let's
+  Encrypt's ACMEv2 API starting on June 19th for the staging environment and
+  July 5th for production. See
+  https://community.letsencrypt.org/t/acmev2-order-ready-status/62866 for more
+  information.
+* Certbot now accepts the flag --reuse-key which will cause the same key to be
```

Comment:
I would probably elaborate just a bit here, to ensure people don't confuse this change with account keys, even though that should be obvious

Reply:
Good idea!